music_assistant/server/providers/deezer/__init__.py

```diff
@@ -530,6 +528,8 @@ def parse_track(
         extra_init_kwargs: dict[str, Any] | None = None,
     ) -> Track | PlaylistTrack:
         """Parse the deezer-python track to a MASS track."""
+        if extra_init_kwargs is None:
+            extra_init_kwargs = {}
         if "position" in extra_init_kwargs:
```

Comment:
This should work

Reply:
yes it is fine now

Reply:
Oh now i changed it to what you proposed..